### PR TITLE
Add Kill extension

### PR DESCRIPTION
**Pull request by benedictdudel**
The kill extension lists all running processes and the user can kill a running process by choosing a process from the list.

### CODE REVIEW

File: Kill.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+__dependencies__ = []
+
+def handleQuery(query):
+    if query.isTriggered:
+        results = []
+        for process in getProcesses(query.string.strip()):
+            results.append(
+                Item(
+                    id=process['id'],
+                    icon=iconLookup(process['name']),
```
Comment (ManuelSchneid3r) [inline Kill.py]: Maybe [*process-stop*](https://standards.freedesktop.org/icon-naming-spec/icon-naming-spec-latest.html#names) makes sense. What do you think? Looks somewhat like this on my system 

image

File: Kill.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                    text=process['name'],
+                    subtext="CPU: %s %%" % process['cpu'],
+                    completion="",
+                    actions=[
+                        ProcAction("Kill", ["kill", "-SIGKILL", process['id']]),
```
Comment (ManuelSchneid3r) [inline Kill.py]: I think sigterm is a better default and as alternative action sigkill

Comment (benedictdudel) [reply Kill.py]: Sounds good :)

File: Kill.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+import os
+import getpass
+from albertv0 import *
+from subprocess import Popen, PIPE
+
+__iid__ = "PythonInterface/v0.1"
+__prettyname__ = "Kill Process"
+__version__ = "1.0"
+__trigger__ = "kill"
```
Comment (ManuelSchneid3r) [inline Kill.py]: Please use space after triggers, otherwise "Killing Hasselhoff" would trigger this extension

**Pull request merged**